### PR TITLE
change init ver, move nor to sub folder, add readme

### PR DESCRIPTION
**Pull request by ginkgm**

### CODE REVIEW

File: esp_flash_nor/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+This component provides some 3rd party NOR flash drivers for ESP-IDF esp_flash driver.
+
+Please be aware of:
+
+- Espressif will do basic reviewing (format, simple logic check, etc.) for the PRs
+- Contributors should do functionality tests on actual flash chips
+- Users should do all neessary tests, including integrated tests, stress tests, stability tests themselves. Users should be responsible for:
+    - Verify 3rd party flash drivers.
+    - Verify related flash models (including their stability, continuous supply) before selecting them into the supply chain. Note supply voltage and temperature may affect a lot to the performance of the flash chips.
```
Comment (Icarus113) [inline esp_flash_nor/README.md:11]: > Note supply voltage and temperature may affect a lot to the performance of the flash chips.

This line works for the root directory README.md as well.

**Review by igrr — COMMENTED** [review]
LGTM, but please move LICENSE file into esp_flash_nor component directory as well.
(The license file has to be present in every component directory.)

**Comment by igrr** [comment]
You can also adjust the component in https://github.com/espressif/esp-flash-drivers/blob/6f55ad80065ccd8902fc177ae489d9077c691bdb/.github/workflows/issue_comment.yml#L18 and the other two Jira-related workflows (E.g. `spi_flash`.) This way the correct component will be set for the Jira issue by default.

File: esp_flash_nor/README.md (hunk 1 above)
Comment (ginkgm) [reply esp_flash_nor/README.md:11]: @Icarus113 As the user may directly use the component without even noticed this repository and the root README.md, I have to add the README to all the components.

But I'll refer to these README.md in the root README

**Pull request merged**